### DISCUSSION TITLE
New Ghostscript version available (9.55.0)

### DISCUSSION THREAD

**Discussion by MattiaCamponesco** [dlemstra/Magick.NET#1064] [General]
Hi everyone!
I was looking into this project today and I find out that there is a new available version of Ghostscript on the official website (9.55.0 - published on 2021-09-27). But it seems that now only the .exe file is downloadable.

In the readme of this project there's a link to Ghostscript (http://www.ghostscript.com/download/gsdnld.html) that is currently unavailable. The latest version of Ghoscript that we have installed on our systems downloaded from the link above is 9.53.3.
Anyway I can confirm that reading of  EPS/PDF/PS  file formats proceed witout errors with the older version.

Are you thinking about support the new version of the tool?

Thanks.

**Comment by dlemstra** [comment]
The ImageMagick library supports the latest version of Ghostscript. You can install and use it. They sometimes make breaking changes that will require a change in ImageMagick but that only happened once in the last 8 years. I will update the link to the correct downloads page later this week, thanks for letting me know.

**Reply by MattiaCamponesco** [reply]
Thank you so much for these clarifications.
The only problem that I noticed is that at the moment from Ghostscript website, it seems that only .exe installer of the application is available. Reading your documentation I see that in the Ghoscript folder of ImageMagick.NET are required both .exe and .dll file (as our currently working setup) but the last one doesn't appear anywhere as downloadable.

**Comment by dlemstra** [comment]
After you run the installer you will get the following files in your system:

```
 Directory of C:\Program Files\gs\gs9.55.0\bin

02/10/2021  17:38    <DIR>          .
02/10/2021  17:38    <DIR>          ..
27/09/2021  09:34        24.698.368 gsdll64.dll
27/09/2021  09:34            11.280 gsdll64.lib
27/09/2021  09:34           238.592 gswin64.exe
27/09/2021  09:34           229.376 gswin64c.exe
               4 File(s)     25.177.616 bytes
               2 Dir(s)  235.977.117.696 bytes free
```

**Reply by MattiaCamponesco** [reply]
Yes, sorry, I didn't remember how we made the last time.
I can confirm that new Ghostscipt version is working well.
Thank you.
